packages/ROBOSID.yaml

```
@@ -1,15 +1,16 @@
 ---
 name: 'ROBOSID'
-version: '1.18a'
-release: 1
+version: '1.8a'
+release: 2
```

In this case, and because the version name changes, the release number should be the same. Release only increases when the version number is the same and only changes in the yaml file are done.
```suggestion
release: 1
```